scripts/sanity_project.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eu
+
+TEST_PROJECT="smoketest"
+
+trap 'catch $? $LINENO' EXIT
+
+catch() {
+  if [[ "$1" != "0" ]]; then
+    echo "Error $1 occurred on $2"
+  fi
+
+  # return to origin, clear directory stack
+  pushd -0 > /dev/null && dirs -c
+
+  if [[ -d "$TEST_PROJECT" ]]; then
+    echo "cleaning up test artifacts"
+    # target directory created in docker has root owner
+    sudo rm -rf "$TEST_PROJECT"
```

Comment:
Note: This is sort of annoying but the docker build context ends up doing everything as root which means the filesystem objects are ultimately owned by root. 

This is worth addressing in another PR.

Reply:
I'm not seeing this issue on macOS (probably due to how Docker volumes just straight up ignore permissions on macOS).

Reply:
fair enough, I'll back it out and leave a comment about how this may run into permission errors on cleanup